test/controllers/regions_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class RegionsControllerTest < ActionController::TestCase
+  setup do
+    @region = regions(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:regions)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create region" do
+    assert_difference('Region.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.